detection-rules/impersonation_github.yml

```diff
@@ -9,9 +9,9 @@ source: |
   and (
       ilike(sender.display_name, '*github*')
       or ilike(sender.email.email, '*github*')
-      or iedit_distance(sender.email.domain.sld, 'github') <= 2
+      or iedit_distance(sender.email.domain.sld, 'github') <= 1
   )
-  and sender.email.domain.root_domain not in ('github.com', 'gitlab.com', 'itthub.net', 'githubsupport.com')
+  and sender.email.domain.root_domain not in ('github.com', 'gitlab.com', 'itthub.net', 'githubsupport.com', 'gtmhub.com')
```

Comment:
I don't think this will ever hit because of the earlier thing, but it doesn't hurt